### DISCUSSION TITLE
sending message to all currently registered workers

### DISCUSSION THREAD

**Discussion by MatteoLacki** [rq/rq#1594] [Q&A]
Hi!

Guys, you have done something magnificent here, many thanks!
I'd like to know, if thereais any way to send one message to a group of specified workers.
I need that for orchestration: basically, I install each worker in a separate virtualenv and I want to send them a message to update  that VE with pip.

Best wishes,

**Comment by ccrvlh** [comment]
@MatteoLacki this looks like a fan out kind of task, I'm not aware of a way to handle that through RQ, maybe with pubsub.
Did you take a look at Redis Stream (or even PubSub) / Kafka to achieve that?

**Reply by MatteoLacki** [reply]
Hmm, you are right: a pubsub could do.

No, I have not looked yet at Redis Stream nor Kafka.

**Reply by ccrvlh** [reply]
I personally would suggest Redis Stream, easy, lightweight, powerful, awesome technology. Worth giving it a shot and should suit your use case.

**Reply by MatteoLacki** [reply]
Yes, I heard about Kafka but it feels like a juggernaut, and I do prefer indeed lightweight, that's why I went for `rq` in the first place.
Thanks a great lot!
